### ISSUE TITLE
Create a Declarative Wrap Component

### ISSUE THREAD

**Issue by jonambas** [SparkPost/matchbox#248]
**Blocked by spacing tokens**

---

Flexbox CSS in the app is growing and should be abstracted into a reusable Matchbox component.

Suggested Props:

**alignment**
vertical alignment -`center` `start` `end` `baseline`

**distribution**
horizontal alignment - `start` `end` `center` `fill`

**orientation**
flex direction - `vertical` `horizontal`
or a boolean `vertical`

**spacing**
enumerable keys of to-be-created spacing tokens that determines spacing between children

**Comment by jonambas** [comment]
Closing in favor of a styled-styem Box component
